test/functional/apps/visualize/_markdown_vis.js

```diff
@@ -34,6 +34,23 @@ export default function ({ getPageObjects }) {
         const actual = await PageObjects.visualize.getMarkdownText();
         expect(actual).to.equal(expected);
       });
+
+      it('should auto apply changes if auto mode is turned on', async function () {
+        const markdown2 = '## Heading 2';
+        await PageObjects.visualize.toggleAutoMode();
+        await PageObjects.visualize.setMarkdownTxt(markdown2);
+        await PageObjects.header.waitUntilLoadingHasFinished();
+        const h1Txt = await PageObjects.visualize.getMarkdownBodyDescendentText('h2');
+        expect(h1Txt).to.equal('Heading 2');
+      });
+
+      it('should resize the editor', async function () {
+        const editorSidebar = await find.byCssSelector('.vis-editor-sidebar');
+        const initialSize = await editorSidebar.getSize();
+        await PageObjects.visualize.sizeUpEditor();
+        const afterSize = await editorSidebar.getSize();
+        expect(afterSize.width).to.equal(initialSize.width + 15);
```

Comment:
Maybe just use `expect(afterSize.width).to.be.greaterThan(initialSize.width);` so we don't make the functional test depending on whatever step size we want to use in the future.